scripts/brp-remove-rpath

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# This script makes sure to remove any standard library search paths from rpath
+# or runpath tags as they're not only redundant, but will break overriding
+# search paths. Search paths of the tag that's not standard will be kept intact.
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+# make sure we have chrpath in our search path, otherwise wuzz out
+type chrpath || exit 0
```

Comment:
I'm not quite sure if this one is a good idea, as there are plenty of cases where RPATH/RUNPATH makes sense to preserve (such as when they are private/plugin libs in `/usr/lib64/<progdir>`).